pyaedt/generic/DataHandlers.py

```diff
@@ -482,6 +487,36 @@ def to_aedt(code):
     return return_code
 
 
+def str_to_bool(s):
+    """Convert a ``"True"`` or ``"False"`` string to its corresponding Boolean value.
+
+    If the passed arguments are not relevant in the context of conversion, the argument
+    itself is returned. This method can be called using the `map()` function to
+    ensure conversion of Boolean strings in a list.
+
+    Parameters
+    ----------
+    s: str
+         Input parameter to convert to a Boolean if the data type is a string
+         with a vlue of ``"True"``, ``"true"``,
+         ``"False"`` or ``"false"``.
```

Comment:
```suggestion
         with a vlue of ``"True"``, ``"true"``, ``"False"``, or ``"false"``.
```
We use Oxford commas, which means a command should always be placed before the "and" or the "or" in a list of items.